### ISSUE TITLE
Error when exporting a plot

### ISSUE THREAD

**Issue by mancap314** [pyqtgraph/pyqtgraph#852]
When I run the [basic export example](http://www.pyqtgraph.org/documentation/exporting.html):

```python
import pyqtgraph as pg
import pyqtgraph.exporters

# generate something to export
plt = pg.plot([1,5,2,4,3])

# create an exporter instance, as an argument give it
# the item you wish to export
exporter = pg.exporters.ImageExporter(plt.plotItem)

# set export parameters if needed
exporter.parameters()['width'] = 100   # (note this also affects height parameter)

# save to file
exporter.export('fileName.png')
```

I get 
```
File "/usr/local/lib/python3.6/dist-packages/pyqtgraph/exporters/ImageExporter.py", line 70, in export
    bg = np.empty((self.params['width'], self.params['height'], 4), dtype=np.ubyte)
TypeError: 'float' object cannot be interpreted as an integer
```

Actually, `self.params['width']` and `self.params['height']` should be casted to `int` on this line

EDIT: I wrote a fix for it locally on a branch but apparently I am not allowed to push it

**Comment by ixjlyons** [comment]
I believe this bug was fixed by #458. Are you still seeing this error using the latest `develop` branch? The fix would not have made it into the latest release.

Also, the typical workflow on GitHub is to fork the repository to your own account, work locally on a new branch, push your local changes to your fork, then submit a pull request.

**Comment by ixjlyons** [comment]
Closing this per my comment above.
